lib/widgets/set_status.dart

```diff
@@ -313,16 +315,9 @@ class StatusSuggestionsListEntry extends StatelessWidget {
 
   @override
   Widget build(BuildContext context) {
-    final designVariables = DesignVariables.of(context);
-
     return InkWell(
       onTap: onTap,
-      splashFactory: NoSplash.splashFactory,
-      overlayColor: WidgetStateColor.resolveWith(
-        (states) => states.any((e) => e == WidgetState.pressed)
-          ? designVariables.contextMenuItemBg.withFadedAlpha(0.20)
-          : Colors.transparent,
-      ),
+      overlayColor: WidgetStatePropertyAll(Colors.transparent),
       child: Padding(
         padding: EdgeInsets.symmetric(vertical: 7, horizontal: 16),
```

Comment:
From the issue:

> * For all page content, take 8px of horizontal external whitespace and move it to the outer `SafeArea.minimum`

But it looks like this commit just adds 8px of horizontal whitespace, without subtracting it from anywhere.